web/docs/setup/deploy/docker.md

```diff
@@ -68,43 +68,43 @@ environment variable by specifying just `-e KEY` without an assignment. Here is
 an example:
 
 ```bash
-docker run -e VAST_ENDPOINT -e VAST_IMPORT__BATCH_SIZE=42 --env-file .env \
-  tenzir/vast start
+docker run -e TENZIR_ENDPOINT -e TENZIR_IMPORT__BATCH_SIZE=42 --env-file .env \
+  tenzir/tenzir start
```

Comment:
Same deal here as above, not sure if this will still be correct.

Reply:
I'll resolve the Docker threads and add a TODO to come back to it after the image situation is cleared up.

Reply:
I'll resolve the Docker threads and add a TODO to come back to it after the image situation is cleared up.